components/text/text.css

```diff
@@ -0,0 +1,45 @@
+/* Headings */
+
+.h1, .h2, .h3, .h4, .h5, .h6 {
```

Comment:
you shouldn't add styles directly to the heading tags. Create a class for it and then apply. By your method you tend to change the default style with is not a good practice

Reply:
Sorry for the confusion but those actually are classes. I just kept the same names.

Reply:
okay cool